### PR TITLE
Add OSM data timestamp to logs

### PR DESCRIPTION
**Pull request by stephankn**
Adding timestamp of OSM data into the logfile to help debugging why some data did not update as expected

### CODE REVIEW

**Comment by stephankn** [comment]
lint complains about a disabled analyzer file I not even touched. 

    analysers/disabled/analyser_merge_wikipedia.py:340: error: unexpected indent

**Comment by frodrigo** [comment]
But we already display the date from the PBF
```
2021-01-23 03:02:34 run pyosmium replication
state:  2021-01-22 03:02:03+00:00 2021-01-23 03:02:34 iteration=1
2021-01-23 03:02:34   2021-01-23 03:02:34,495 INFO: Using replication service at http://download.openstreetmap.fr/replication/./africa/algeria/minute
2021-01-23 03:02:34   2021-01-23 03:02:34,886 INFO: Starting download at ID 4381689 (max 1024 MB)
2021-01-23 03:03:35   2021-01-23 03:03:35,572 INFO: Downloaded until 4383128. Server has data available until 4383128.
state:  1  -  2021-01-23 03:02:03+00:00     -1 day, 0:00:00
2021-01-23 03:03:35   DROP SCHEMA algeria
2021-01-23 03:03:35 import osmosis schema
```

**Comment by stephankn** [comment]
@frodrigo I think this only happens when you update a diff. Downloading pbf is not printing it

**Comment by frodrigo** [comment]
> @frodrigo I think this only happens when you update a diff. Downloading pbf is not printing it

You are right, this part is only on update.

File: modules/OsmOsisManager.py [hunk 1]
```diff
@@ -213,6 +213,7 @@ def update_metainfo(self, conf):
     osm_state = OsmPbfReader(dst_pbf, state_file=state_file).timestamp()
 
     giscurs.execute("UPDATE metainfo SET tstamp = %s", [osm_state])
+    self.logger.sub().log("OSM data timestamp: %s" % osm_state)
```
Comment (frodrigo) [inline modules/OsmOsisManager.py:216]:
```suggestion
    self.logger.sub().log("OSM data timestamp: {}".format(osm_state))
```

**Comment by frodrigo** [comment]
Ok. Merged.

**Pull request merged**